.travis.yml

```diff
@@ -9,8 +9,8 @@ cache:
 matrix:
     fast_finish: true
     include:
-        - php: 7.0
-        - php: 7.0
+        - php: '7.0'
```

Comment:
But its *their* parser, they have control on it. For example PHP Yaml parser would give the same result, proof is SensioLabsInsight complains about `.travis.yml` being incorrect (although Traivis' YAML parser understand it).

Anyway safer the better :)